README.md

```diff
@@ -78,7 +78,7 @@ window.imagePicker.getPictures(
     
 ### Note for Android Use
 
-When outputType is FILE_URI the plugin returns images that are stored in a temporary directory.  These images will often not be deleted automatically though.  The files should be moved or deleted after you get their filepaths in javascript. If Base64 Strings are being returned, there is nothing to clean up.
+The 'maximumImagesCount' option is currently the only option supported for Android
```

Comment:
- [ ] Changelog - breaking change because of play store policy update
- [ ] Options supported
  - maximumImagesCount but only toast shown for now
- [ ] New package name